packages/block-editor/src/components/inserter/block-types-tab.js

```diff
@@ -43,22 +43,14 @@ export function BlockTypesTab( {
 	}, [ items ] );
 
 	const itemsPerCategory = useMemo( () => {
-		const getCategoryIndex = ( item ) => {
```

Comment:
This seems to do nothing useful. I don't think I'm missing something here, but I'l like a sanity check :)

Reply:
The original intent for this was to sort the categories, though they might be sorted elsewhere now :)